webui/src/layout/Header/Header.tsx

```diff
@@ -1,32 +1,29 @@
-import React from "react";
-import styles from "./Header.module.scss";
-import { ReactComponent as BacalhauLogo } from "../../images/bacalhau.svg";
-import { ReactComponent as ProfileIcon } from "../../images/profile.svg";
+import React from "react"
+import { SVGImage } from "../../images/svg-image"
+import styles from "./Header.module.scss"
 
 interface HeaderProps {
-  collapsed: boolean;
-  pageTitle: string;
+  collapsed: boolean
+  pageTitle: string
 }
 
-const Header: React.FC<HeaderProps> = ({ pageTitle, collapsed }) => {
-  return (
-    <header className={styles.header} data-collapsed={collapsed}>
-      <div className={styles.left}>
-        <BacalhauLogo className={styles.logo} height="24" />
-        <div className={styles.pageTitle}>{pageTitle}</div>
-        <div className={styles.searchBar}>
-          {/* Placeholder for search bar */}
-          {/* <input type="text" placeholder="Search..." /> */}
-        </div>
+export const Header: React.FC<HeaderProps> = ({ pageTitle, collapsed }) => (
+  <header className={styles.header} data-collapsed={collapsed}>
+    <div className={styles.left}>
+      <SVGImage src="../../images/bacalhau.svg" alt="Bacalhau Icon" />
```

Comment:
As I wrote in the previous PR, you are using `SVGImage` here and the old style `... as ReactComponent` in different places in the code. Please can you explain why or unify the two methods to a single approach.

Reply:
yes - because i had several requests to break up the PRs - the PR you mention about using the correct type (SVGImage) instead of the ReactComponent is in a later PR. However, every so often later PRs leak through.